stream.go

```diff
@@ -114,7 +114,7 @@ func (cc *ClientConn) NewStream(ctx context.Context, desc *StreamDesc, method st
 }
 
 // NewClientStream creates a new Stream for the client side. This is typically
```

Comment:
How about

```go
// NewClientStream is a wrapper for ClientConn.NewStream.
//
// DEPRECATED: ....
```

to avoid the need to duplicate documentation.

Reply:
Done